api/dbadapters/execution_sql_adapter.ts

```diff
@@ -0,0 +1,303 @@
+import * as semver from "semver";
+import { Task, Tasks, concatenateQueries } from "df/core/tasks";
+import { dataform } from "df/protos/ts";
+import { tableTypeEnumToString } from "df/core/utils";
+import { ErrorWithCause } from "df/common/errors/errors";
+import { CompilationSqlAdapter } from "df/core/compilation_sql_adapter";
+
+export type QueryOrAction = string | dataform.Table | dataform.Operation | dataform.Assertion;
+
+export interface IValidationQuery {
+  query?: string;
+  incremental?: boolean;
+}
+
+export class ExecutionSqlAdapter {
```

Comment:
nit: these aren't really Adapters anymore, it's just a ... helper?

Reply:
Have removed that suffix altogether!